src/systemctl/systemctl-list-jobs.c

```diff
@@ -39,7 +39,7 @@ static int output_waiting_jobs(sd_bus *bus, Table *table, uint32_t id, const cha
                                     TABLE_EMPTY,
                                     TABLE_EMPTY);
                 if (rc < 0)
-                        return table_log_add_error(r);
+                        return table_log_add_error(rc);
```

Comment:
Hmm, there's actually no need for a separate variable. Please just store the retval table_add_many in r instead.